api/datastore/migrator/migrate_dbversion34_test.go

```diff
@@ -0,0 +1,94 @@
+package migrator
+
+const (
+	db35TestFile = "portainer-mig-35.db"
+	username     = "portainer"
+	password     = "password"
+)
+
+// TODO: this is exactly the kind of reaching into the internals of the store we should not do
```

Comment:
this file is commented, can we remove or fix?

Reply:
this is something @hookenz probably needs to help me with later - i commented it out because i ran out of rebase energy after about 3.5 hours yesterday

Reply:
Basically creates an old style Dockerhub authentication and transitions it to the new.